SPECS/aide/aide-0.15-syslog-format.patch

```diff
@@ -1,496 +0,0 @@
-diff -up ./doc/aide.conf.5.in.syslog_format ./doc/aide.conf.5.in
```

Comment:
Are these patches already applied in 0.18.6 ?

Reply:
No, I'll remove it. Seems like an oversight from my side